src/main/java/com/amazon/opendistroforelasticsearch/jobscheduler/sweeper/JobSweeper.java

```diff
@@ -188,16 +188,16 @@ public void postIndex(ShardId shardId, Engine.Index index, Engine.IndexResult re
 
         ShardNodes shardNodes = new ShardNodes(localNodeId, shardNodeIds);
         if (shardNodes.isOwningNode(index.id())) {
-            this.sweep(shardId, index.id(), result.getVersion(), index.source());
+            this.sweep(shardId, index.id(), result.getVersion(), index.source(), result.getTerm(), result.getSeqNo());
```

Comment:
Can we pass in the JobDocVersion instead of all 3 properties separately to reduce parameters

Reply:
refactored to pass JobDocVersion